litmus-portal/frontend/src/views/WorkflowDetails/WorkflowStatus/styles.ts

```diff
@@ -18,6 +18,18 @@ const useStyles = makeStyles((theme) => ({
     color: theme.palette.status.completed.text,
   },
 
+  error: {
+    color: '#FFA600',
+  },
+
+  skipped: {
+    color: '#0098DD',
+  },
+
+  omitted: {
+    color: '#A93DDB',
+  },
+
```

Comment:
We need to adhere to the Litmus UI theme colors, we can either add these colors or change them to error success and pending.

Reply:
**Took a look at Special Notes**